.github/workflows/trivy.yaml

```
@@ -0,0 +1,105 @@
+            - CVE-2025-24928
+            - CVE-2025-27113
+          exit-code: 1
+
+      - name: Scan backend image
+        uses: aquasecurity/trivy-action@6c175e9c4083a92bbca2f9724c8a5e33bc2d97a5
+        with:
+          image-ref: ${{ env.DOCKERHUB_USERNAME }}/owasp-nest-backend:staging
+          severity: 'HIGH,CRITICAL'
+          ignore-unfixed: true
+          ignore-cves:
+            - CVE-2025-31115
```

actionlint (1.7.4)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)